src/legacy/core_plugins/data/public/search/aggs/index.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export { aggTypes } from './agg_types';
+export { AggType } from './agg_type';
+export { AggConfig } from './agg_config';
+export { AggConfigs } from './agg_configs';
+export { FieldParamType } from './param_types';
+export { MetricAggType } from './metrics/metric_agg_type';
+export { AggTypeFilters } from './filter';
+export { aggTypeFieldFilters, AggTypeFieldFilters } from './param_types/filter';
+export {
+  parentPipelineAggHelper,
+  parentPipelineType,
+} from './metrics/lib/parent_pipeline_agg_helper';
+export {
+  siblingPipelineAggHelper,
+  siblingPipelineType,
+} from './metrics/lib/sibling_pipeline_agg_helper';
+
+// static code
+export { AggParamType } from './param_types/agg';
```

Comment:
We decided to create a namespace for static export  ;)

Reply:
My plan was to revisit the actual structure of the contracts for this service in a follow-up PR, where I'd update the structure of both static & runtime contracts at once.

Also Liza discovered a potential issue with namespaces & generated documentation. [It is being discussed here](https://github.com/elastic/kibana/issues/52374#issuecomment-581490235) whether they will still be a viable option for us, so if we have an answer to that soon I can update the implementation accordingly.